src/python/tools/db2graph/db2graph.py

```diff
@@ -0,0 +1,712 @@
+        cursor = []
+        if db_server == 'maria-db':
+            cursor = cnx.cursor()
+        elif db_server == 'postgre-sql':
+            cursor = cnx.cursor(name = cursor_name)
+        cursor.execute(query)
+        logging.info(f'Cursor.execute time is: {time.time() - add_time1}')
+            
+        # Getting Basic Details
+        table_name_list = re.split(' ', query)  # table name of the query to execute
+        table_name1 = table_name_list[1].split('.')[0] # src table
+        col_name1 = table_name_list[1].split('.')[1][:-1] # src column, (note last character ',' is removed)
+        
+        # Processing each batch of cursor on client
+        rowsCompleted = 0
+
+        # In an initial sample pass, estimates the optimal maximum possible fetchSize for given query based on memory usage report of virtual_memory() 
+        # process data with fetchSize=10000, record the amount of memory used,
+        # increase fetchSize if the amount of memory used is less than half of machine's total available memory, 
+        # Note: all unit size are in bytes, fetchSize limited between 10000 and 100000000 bytes
+        if first_pass:
+            mem_copy = psutil.virtual_memory()
+            mem_copy_used = mem_copy.used
+            limit_fetchSize = min(mem_copy.available / 2, 1000000000) # max limit 1 billion
+
+        # Potential issue: There might be duplicates now possible as drop_duplicates over smaller range
+        # expected that user db does not have dupliacted
+        while (True): # Looping till all rows are completed and processed
+            result = cursor.fetchmany(fetchSize)
+            result = pd.DataFrame(result)
+            if (result.shape[0] == 0):
+                break
+
+            # Cleaning Part
+            result = result.applymap(clean_token)  # strip tokens and lower case strings
+            result = result[~result.iloc[:, 1].isin(INVALID_ENTRY_LIST)]  # clean invalid data
+            result = result[~result.iloc[:, 0].isin(INVALID_ENTRY_LIST)]
+            result = result.drop_duplicates()  # remove invalid row
+
+            result.iloc[:, 0] = table_name1 + "_" + col_name1 + '_' + result.iloc[:, 0]   # src
+            result.insert(1, "rel", edge_entity_feature_val_rel_list[i])  # rel
+            result.columns = ["src", "rel", "dst"]
+            
+            if generate_uuid:
+                # convert entity nodes to respective UUIDs
+                result['src'] = result['src'].map(entity_mapping)
+                if (result['src'].isna().any()):
+                    logging.warning(f'Some src column entities did not map in edges_entity_entity')
+                    exit(1)
+
+            result.to_csv(output_dir / Path("all_edges.txt"), sep='\t',\
+                header=False, index=False, mode='a') # Appending the output to disk
+            del result
+            rowsCompleted += fetchSize
```

Comment:
Nit: be consistent with variable naming. Python variables should be snake case.

So:
rowsCompleted -> rows_completed
fetchSize -> fetch_size

Make the necessary changes here and elsewhere